remove_videos_dl_in_other_languages/instructions.txt

```diff
@@ -0,0 +1,6 @@
+If your primary language is English and the courses downloaded by the script came in multiple languages, you can delete them by doing this:
+
+1. Grant executable permissions to the remove_extra_lang.sh:
+   chmod +x remove_extra_lang.sh
+2. Execute it on the folder in which you would like to delete the extra videos:
+   ./remove_extra_lang.sh
```

Comment:
If our hook methods are working, you may want to put instructions on how to use your script with the `--hook` option to automate the process.